### ISSUE TITLE
Extra Utilities and NEI?

### ISSUE THREAD

**Issue by alxd133** [rwtema/extrautilities_old_issues#412]
I need some help...

I've been trying to put together a small private modpack with just a few mods that I like to play, that don't require a beast of a PC. I've gotten everything to work fairly easily, but now I can't figure out why these mods won't work. I am trying to install OpenBlocks and Extra Utils. Neither of them will work. I've narrowed it down to where if Extra Utils or Openblocks is sepparated from NEI, they will load. But as soon as a put NEI back in, it gives a few lines of red code in the console (I'm using MultiMC to launch), then says, 'Minecraft exited with exitcode 1.'

http://pastebin.com/xjsF0sxv  - MultiMC rport

http://pastebin.com/iBTQAggz  - Minecraft crash report

List of current mods:
AOBD
AE2
Backpack
BetterStorage
CodeChickenCore
ExtraTic
JABBA
Mantle
Metallurgy 4
Metallurgy Core
NEI
TiC Tooltips
Tinker's Construct
WAILA

**Comment by TherminatorX** [comment]
NEI and WAILA are client-side only mods. Remove them. If it still won't work, remove TiC Tooltips.

**Comment by alxd133** [comment]
TherminatorX, as I said in the original comment, it works if I take off NEI. But that's why I brought the problem here, I need it to work with NEI and OpenBlocks.

I tested it without NEI, OpenBlocks, TiC Tooltips, and WAILA. It's working.

Then I re-enabled NEI. It launched, black screened, and then gave me this; http://pastebin.com/Qc10164L

Tested without NEI, OpenBlocks, or WAILA. It worked with TiC tooltip. (Tried with just TiC tooltip and WAILA, but WAILA is an add-on for NEI)

Tested without NEI and WAILA, same 'Minecraft exited with exitcode 1'

**Comment by Mengmoshu** [comment]
NEI and WAILA both support being installed on the server. WAILA for example can be put on the server to allow admins to disable some of the cheatier features. I'm not sure what all NEI does when it's on the server, but it's been a server friendly mod for ages.

**Comment by alxd133** [comment]
Mengmoshu, it's not for a server. It's a private modpack I'm building.

**Comment by Mengmoshu** [comment]
I was responding to @TherminatorX stating that those 2 mods were clientside only. They work in a client connecting to servers without them, but they are actually intended to work on a server. Sadly I don't think that info really matters to your problem, sorry I can't help with that.

**Comment by tterrag1098** [comment]
More than that, WAILA and NEI _need_ to be on the server for some features to work. Without WAILA on the server it cannot sync NBT values and a lot of plugins will break.

**Comment by TherminatorX** [comment]
How did OpenBlocks get in this conversation?

Also, I realized I hadn't read the crashlogs at all, and:
ExtraUtilities{1.1.0k} [Extra Utilities](extrautilities-1.1.0k.jar) Unloaded->Constructed->Errored
and
cpw.mods.fml.common.LoaderException: java.lang.NoClassDefFoundError: com/rwtema/extrautils/item/ItemHeatingCoil

That doesn't look good.

**Comment by alxd133** [comment]
Yeah, I got it to work with OpenBlocks and NEI, now it will even launch. It black sreens then gives me this; http://pastebin.com/VXNKpFBh

**Comment by TherminatorX** [comment]
I guess it may have something to do with NEI and the heating coil. Perhaps it's that the hc doesn't get consumed when smelting, and NEI gets confused. @rwtema is the one with the code, so we'll have to see.

By the way, the first 55 lines are the same, minus the time and witty comment.
